features/step_definitions/admin_steps.rb

```diff
@@ -138,15 +138,15 @@
 Given /^the support form is disabled and its text field set to "Please don't contact us"$/ do
   step("I am logged in as support_admin")
   visit(admin_settings_path)
-  check("Turn off support form")
+  check("Turn off Support form")
```

Comment:
Hmm, should it be? 🤔 I think we're using "support' in the generic sense here, i.e. it's the support and feedback form, as opposed to the Support Committee form.